app/assets/javascripts/admin/addon/mixins/setting-component.js

```diff
@@ -204,10 +207,59 @@ export default Mixin.create({
     }
   },
 
+  async confirmChanges(settingKey) {
+    return new Promise((resolve) => {
```

Comment:
you should just await this instead of returning, async will always return a promise

Reply:
Ah nice, will change